tron/trondaemon.py

```diff
@@ -85,7 +85,7 @@ def run(self):
                 signal.SIGUSR1: self._handle_debug,
             }
             signal.pthread_sigmask(signal.SIG_BLOCK, signal_map.keys())
-
+            self._run_rookout()
```

Comment:
it'd be nice to wrap this with a try-except just to ensure that any issues with rookout/rookout setup don't stop tron from starting